### ISSUE TITLE
can't use in flutter 1.12.13

### ISSUE THREAD

**Issue by w8023w1314** [rmawatson/flutter_isolate#41]
when i use sqflite api in isolate , report some error :
E/FlutterIsolate(31605): NoSuchMethodException: io.flutter.plugins.GeneratedPluginRegistrant.registerWith [interface io.flutter.plugin.common.PluginRegistry]
E/FlutterIsolate(31605): The plugin registrant must provide a static registerWith(FlutterPluginRegistry) method

but, i change flutter v1.9.x  , not this error

**Comment by w8023w1314** [comment]
https://www.jianshu.com/p/7b551ef85e3c

this has the same error , but i cant understand how to fix it

**Comment by rohansohonee** [comment]
I am also facing the same issue

**Comment by w8023w1314** [comment]
https://www.jianshu.com/p/01bab96dfe79
flutter  1.9.1 -->  1.12.13  

在 Flutter 1.12 开始 Flutter 团队调整了 Android 插件的实现代码，在 1.12 之后 Android 开始使用新的插件 API ，基于的旧的 PluginRegistry.Registrar 不会立即被弃用，但官方建议迁移到基于的新API FlutterPlugin

**Comment by w8023w1314** [comment]
i think the flutter_isolate plugin  need upgrade .  有没有大神帮忙解决一下

**Comment by rohansohonee** [comment]
It needs to implement the v2 embedding

**Comment by w8023w1314** [comment]
> It needs to implement the v2 embedding

so , can you resolve it ?   wechat      +86 17688898884    ,  我就是个小菜鸡呀. 怎么会遇到这种难题

**Comment by rohansohonee** [comment]
Check out this [Plugin migration guide](https://flutter.dev/docs/development/packages-and-plugins/plugin-api-migration) . 
>so , can you resolve it ?

I will give it a try.

**Comment by w8023w1314** [comment]
> Check out this [Plugin migration guide](https://flutter.dev/docs/development/packages-and-plugins/plugin-api-migration) .
> 
> > so , can you resolve it ?
> 
> I will give it a try.

thx , if you resolve it , please tell me ~   you'll my  Saviour

**Comment by rohansohonee** [comment]
[Link to my pull request](https://github.com/rmawatson/flutter_isolate/pull/42)
If you want to use the new v2 embedding you can clone my repo as well from [here](https://github.com/rohansohonee/flutter_isolate)

If you encounter any issues do let me know.

**Comment by ryanheise** [comment]
It's now merged. @w8023w1314 , does git master work for you? i.e. in your pubspec.yml try:

```yml
  flutter_isolate:
    git:
      url: git@github.com:rmawatson/flutter_isolate.git
```

**Comment by w8023w1314** [comment]
it's work , thx~              顺颂夏祺

**Comment by ryanheise** [comment]
This is now published as 1.0.0+12
